### PR TITLE
KAFKA-4244: fixing formating issues in docs. missing headers and lots of paragrap…

### PR DESCRIPTION
**Pull request by gwenshap**
…h misformatting

### CODE REVIEW

File: docs/protocol.html [hunk 1]
```diff
@@ -16,8 +16,11 @@
 -->
 
 <!--#include virtual="../includes/header.html" -->
-
-<h3><a id="protocol" href="#protocol">Kafka Wire Protocol</a></h3>
+<!--#include virtual="../includes/top.html" -->
+<div class="content">
+    <!--#include virtual="../includes/nav.html" -->
+    <div class="right">
```
Comment (hachikuji) [inline docs/protocol.html:22]: Forgive my poor html knowledge, but do these `div` blocks need to end somewhere?

**Comment by hachikuji** [comment]
@gwenshap Had one question, but otherwise LGTM.

File: docs/protocol.html (hunk 1 above)
Comment (gwenshap) [reply docs/protocol.html:22]: I asked Derrick same thing! Apparently they close in the footer :)

Comment (hachikuji) [reply docs/protocol.html:22]: Ok, works for me.

File: docs/quickstart.html [hunk 2]
```diff
@@ -167,13 +180,14 @@
...
 
-Now let's test out fault-tolerance. Broker 1 was acting as the leader so let's kill it:
+<p>Now let's test out fault-tolerance. Broker 1 was acting as the leader so let's kill it:</p>
 <pre>
 &gt; <b>ps aux | grep server-1.properties</b>
 <i>7564</i> ttys002    0:15.91 /System/Library/Frameworks/JavaVM.framework/Versions/1.8/Home/bin/java...
 &gt; <b>kill -9 7564</b>
 </pre>
 
+<<<<<<< HEAD
```
Comment (hachikuji) [inline docs/quickstart.html]: Looks like you missed one conflict while rebasing.

File: docs/quickstart.html [hunk 3]
```diff
@@ -92,15 +95,22 @@
 
 <h4><a id="quickstart_multibroker" href="#quickstart_multibroker">Step 6: Setting up a multi-broker cluster</a></h4>
 
-So far we have been running against a single broker, but that's no fun. For Kafka, a single broker is just a cluster of size one, so nothing much changes other than starting a few more broker instances. But just to get feel for it, let's expand our cluster to three nodes (still all on our local machine).
+<p>So far we have been running against a single broker, but that's no fun. For Kafka, a single broker is just a cluster of size one, so nothing much changes other than starting a few more broker instances. But just to get feel for it, let's expand our cluster to three nodes (still all on our local machine).</p>
 <p>
+<<<<<<< HEAD
```
Comment (hachikuji) [inline docs/quickstart.html]: Woops, here too.

File: docs/introduction.html [hunk 4]
```diff
@@ -14,156 +14,184 @@
...
 It isn't enough to just read, write, and store streams of data, the purpose is to enable real-time processing of streams.
+</p>
 <p>
 In Kafka a stream processor is anything that takes continual streams of  data from input topics, performs some processing on this input, and produces continual streams of data to output topics.
+</p>
 <p>
 For example a retail application might take in input streams of sales and shipments, and output a stream of reorders and price adjustments computed off this data.
+</p>
 <p>
+<<<<<<< HEAD
```
Comment (hachikuji) [inline docs/introduction.html:167]: Sorry, missed this one on my last pass.

**Comment by hachikuji** [comment]
LGTM. I'll fix the conflict tag when I merge.

**Pull request closed without merging**